gloo/docs/installation/gateway/kubernetes/_index.md

```diff
@@ -1,193 +1,85 @@
-* [Install via Helm](#install-gloo-with-helm)
-
-### Verify Installation
-
-* [Verify installation](#verify-your-installation)
-
-### Uninstall Gloo
-
-* [Uninstall Gloo](#uninstall)
-
-We highly recommend installing Gloo using the Gloo CLI as it simplifies a lot of the user experience of using Gloo.
-For power users, feel free to use the underlying `yaml` configuration files directly.
-
-These directions assume you've prepared your Kubernetes cluster appropriately. Full details on setting up your
-Kubernetes cluster [here](../setup_kubernetes).
-
-There are several options for deploying Gloo, depending on your use case and deployment platform.
-
-* **Gateway**: (**recommended**) Gloo's full feature set is available via its v1/Gateway API. The Gateway API
-is modeled on Envoy's API with the use of opinionated defaults to make complex configurations possible
-while maintaining simplicity when required.
-
-* **Ingress**: Gloo supports the Kubernetes Ingress resource, acting as a Kubernetes Ingress Controller.
-
-{{% notice note %}}
-ingress objects must have the annotation `"kubernetes.io/ingress.class": "gloo"` to be processed by the Gloo Ingress.
-{{% /notice %}}
-
-* **Knative**: Gloo integrates automatically with Knative as a cluster-level ingress for
-[*Knative-Serving*](https://github.com/knative/serving). Gloo can be used in this way as a lightweight replacement
-for Istio when using Knative-Serving.
-
-{{% notice info %}}
-If this process does not work, please [open an issue](https://github.com/solo-io/gloo/issues/new).
-We are happy to answer questions on our [diligently staffed Slack channel](https://slack.solo.io/) as well.
-{{% /notice %}}
-
----
-
-## Install Gloo via Command Line Interface (CLI)
-
-Choosing a deployment option for installing Gloo into your Kubernetes cluster:
+```shell
+brew install solo-io/tap/gloo
+```
 
-* [Gateway](#install-the-gloo-gateway-to-your-kubernetes-cluster-using-glooctl) (**recommended**)
-* [Ingress](#install-the-gloo-ingress-controller-to-your-kubernetes-cluster-using-glooctl)
-* [Knative](http://localhost:1313/installation/kubernetes/quick_start/#install-the-gloo-knative-cluster-ingress-to-your-kubernetes-cluster-using-glooctl)
+To install on any platform run the following.
 
-{{% notice info %}}
-You can install Gloo to an existing namespace by providing the `-n` option, e.g. `glooctl install gateway -n my-namespace`.
-If the option is not provided, the namespace defaults to `gloo-system`.
-{{% /notice %}}
+```bash
+curl -sL https://run.solo.io/gloo/install | sh
+```
 
-### Install the Gloo Gateway to your Kubernetes Cluster using `glooctl`
+You can download `glooctl` directly via the GitHub releases page. You need to add `glooctl` to your path after downloading.
 
-Once your Kubernetes cluster is up and running, run the following command to deploy the Gloo Gateway to the `gloo-system` namespace:
 
 ```bash
-glooctl install gateway
+export PATH=$HOME/.gloo/bin:$PATH
```

Comment:
This applies just to the `curl` install path, but since the preceding sentence is the "download from github" option, it might be confusing for the user (e.g. where does that `.gloo/bin` directory come from?)